client/my-sites/plugins/plugin-meta/index.jsx

```diff
@@ -430,27 +430,35 @@ class PluginMeta extends Component {
 	}
 
 	render() {
+		const {
+			selectedSite,
+			plugin: pluginProp,
+			isPlaceholder,
+			sites,
+			atEnabled,
+			slug,
+			translate,
+			isMock,
+		} = this.props;
 		const cardClasses = classNames( 'plugin-meta__information', {
 			'has-button': this.hasOrgInstallButton(),
-			'has-site': !! this.props.selectedSite,
-			'is-placeholder': !! this.props.isPlaceholder
+			'has-site': !! selectedSite,
+			'is-placeholder': !! isPlaceholder
 		} );
-
-		const plugin = this.props.selectedSite && this.props.sites[ 0 ] && this.props.sites[ 0 ].plugin
-			? this.props.sites[ 0 ].plugin
-			: this.props.plugin;
-		const path = ( ! this.props.selectedSite || plugin.active ) && this.getExtensionSettingsPath( plugin );
+		const selectedSitePlugin = get( sites, '[0].plugin', pluginProp );
+		const plugin = selectedSite ? selectedSitePlugin : pluginProp;
```

Comment:
You're right. I was trying to make the logic simpler by breaking it to two lines. But I think I've just made it more complex. I pushed up a change that keeps the logic on one line to help avoid the need to create a new variable name. Lemme know what'cha think.